moped-database/README.md

```diff
@@ -94,16 +94,16 @@ Configure your environment:
    $ cp env_template env
 ```
 
-2. Pull a copy of production data and start the cluster. This will get a new copy of production (once per day) and store the snapshot in the `./snapshots/` folder. Subsequent invocations of this command on the same day will not download a new copy. 
+2. Pull a copy of production data, start the cluster, and apply local migrations and metadata. This will get a new copy of production (once per day) and store the snapshot in the `./snapshots/` folder. Subsequent invocations of this command on the same day will not download a new copy. 
 
 ```bash
 $ ./hasura-cluster replicate
 ```
 
-If you would prefer to bypass downloading a new copy of production and instead just use whatever the last snapshot in the folder, add `--use-latest-snapshot` or `-j` 
+If you would prefer to bypass downloading a new copy of production and instead just use whatever the last snapshot in the folder, add `--use-any-snapshot` or `-j` 
 
 ```bash
-$ ./hasura-cluster --use-latest-snapshot
+$ ./hasura-cluster --use-any-snapshot
```

Comment:
@johnclary thanks for updating all of this 🙏 can you update this line to include the replicate part of the command?

Reply:
yes—thanks!